workbench-core/authentication/src/plugins/cognitoAuthenticationPlugin.ts

```diff
@@ -400,35 +399,20 @@ export class CognitoAuthenticationPlugin implements AuthenticationPlugin {
     const describeCommand = new DescribeUserPoolClientCommand(describeInput);
 
     try {
-      const clientInfo = await client.send(describeCommand);
-
-      const {
-        IdToken: idTokenUnits,
-        AccessToken: accessTokenUnits,
-        RefreshToken: refreshTokenUnits
-      } = clientInfo.UserPoolClient?.TokenValidityUnits || {};
-
-      const refreshTokenTime = clientInfo.UserPoolClient?.RefreshTokenValidity;
-      const idTokenTime = clientInfo.UserPoolClient?.IdTokenValidity;
-      const accessTokenTime = clientInfo.UserPoolClient?.AccessTokenValidity;
-
-      const idTokenExpiresIn =
-        idTokenTime && idTokenUnits
-          ? getTimeInSeconds(idTokenTime, idTokenUnits as TimeUnitsType)
-          : undefined;
-      const accessTokenExpiresIn =
-        accessTokenTime && accessTokenUnits
-          ? getTimeInSeconds(accessTokenTime, accessTokenUnits as TimeUnitsType)
-          : undefined;
-      const refreshTokenExpiresIn =
-        refreshTokenTime && refreshTokenUnits
-          ? getTimeInSeconds(refreshTokenTime, refreshTokenUnits as TimeUnitsType)
-          : undefined;
+      const { UserPoolClient } = await client.send(describeCommand);
```

Comment:
is this cognito API expected to be hit at a high TPS? Should we cache this result and get it a different way?

Reply:
It is hit on login and on refreshing access tokens, so once every ~15 minutes per user if access tokens have a 15 minute lifespan.